### ISSUE TITLE
debug_traceBlockByHash returns error after 1.10.1

### ISSUE THREAD

**Issue by jayuuza** [ethereum/go-ethereum#22675]
#### System information

Geth version: v1.10.1
OS & Version: OSX

#### Expected behaviour

Return traced data

#### Actual behaviour

Returns {code=-32000, gas=229982, hash=0x32b61c4545d6d0bbef9010145fb8bed6179848c59a2964c8c0b781af24e343a9, height=12244400,message=tracing failed: insufficient funds for transfer: address 0xC585AE55a9878BF5b0DF95d88c1588AeEAda7C03}

#### Steps to reproduce the behaviour

Call debug_traceBlockByHash on block 12244400
0xBAD5B0
0x32b61c4545d6d0bbef9010145fb8bed6179848c59a2964c8c0b781af24e343a9

Example:
```
curl --cookie "$COOKIE" -H "Content-Type: application/json" \
  --data '{"method":"debug_traceBlockByHash","params":["0x32b61c4545d6d0bbef9010145fb8bed6179848c59a2964c8c0b781af24e343a9",{"tracer":"callTracer","timeout":"100s"}],"id":557700677779410}' \
  node_url {"jsonrpc":"2.0","id":557700677779410,"error":{"code":-32000,"message":"insufficient funds for transfer: address 0xC585AE55a9878BF5b0DF95d88c1588AeEAda7C03"}}

```

#### Backtrace

````
WARN [04-15\|11:33:03.662] Served debug_traceBlockByHash            conn=172.30.32.74:44508  reqid=6628171866060648146 t=4.83987438s    err="insufficient funds for transfer: address 0xC585AE55a9878BF5b0DF95d88c1588AeEAda7C03"
````

**Comment by alexqrid** [comment]
Solved this just by upgradig to `v1.10.2`.

**Comment by jayuuza** [comment]
Thanks @alexqrid seems to have helped. I think this https://github.com/ethereum/go-ethereum/issues/22485 was the fix
